python/heterocl/tvm/build_module.py

```diff
@@ -455,10 +455,10 @@ def build_fpga_kernel(sch, args, target, name="default_function"):
         if "|" in mode:
             modes = mode.split("|")
             for m in modes:
-                assert m in ["csyn", "csim", "cosim", "impl"], \
+                assert m in ["csyn", "csim", "cosim", "impl", "customized"], \

```

Comment:
Not sure if we need the "customized" mode here.

Reply:
If "customized" mode is not added here, users need to explicitly set the mode (csyn, csim, cosim, impl) even when they pass in custom Tcl script. However, we cannot know what is done in user's script.